frontend/src/app/modules/global_search/input/global-search-input.component.ts

```diff
@@ -425,6 +425,13 @@ export class GlobalSearchInputComponent implements OnInit, OnDestroy {
     (<HTMLInputElement>document.activeElement).blur();
   }
 
+  public statusHighlightClass(statusId:any) {
+    if (!statusId) {
+      return;
+    }
+    return Highlighting.backgroundClass('status', statusId!);
```

Comment:
There is an a method in the `global-search-input.component` called `statusHighlighting`. It uses the `inlineClass` as opposed to the backgroundClass as done here.

Personally, I find the bubble in front of the status way more readable but it is probably personal preference. 

In any case, one of the methods should be removed.

Reply:
Hi @ulferts thanks for your feedback, I agree with you.